src/react-app-env.d.ts

```diff
@@ -2,3 +2,6 @@
 declare module 'carbon-icons';
 declare module 'react-blockies';
 declare module 'react-jazzicon';
+interface Window {
+  ethereum: any;
```

Comment:
I did it like this for a long time but I just checked and apparently there's finally good typings you can put here: https://stackoverflow.com/a/70074252 :)